lib/awskeyring_command.rb

```diff
@@ -104,11 +104,16 @@ def list_role
   desc 'env ACCOUNT', I18n.t('env_desc')
   method_option 'no-token', type: :boolean, aliases: '-n', desc: I18n.t('method_option.notoken'), default: false
   method_option :unset, type: :boolean, aliases: '-u', desc: I18n.t('method_option.unset'), default: false
+  method_option :force, type: :boolean, aliases: '-f', desc: I18n.t('method_option.force'), default: false
   # Print Env vars
   def env(account = nil)
     if options[:unset]
       put_env_string(account: nil, key: nil, secret: nil, token: nil)
     else
+      if $stdout.isatty && !options[:force]
```

Comment:
This will be false if:
awskeyring | cat

Is that acceptable behaviour?

Reply:
yes